cognite_toolkit/_cdf_tk/utils/dtype_conversion.py

```diff
@@ -0,0 +1,266 @@
+import ctypes
+import json
+from abc import ABC, abstractmethod
+from collections.abc import Mapping
+from typing import ClassVar, cast
+
+from cognite.client.data_classes.data_modeling.data_types import Enum, ListablePropertyType
+from cognite.client.data_classes.data_modeling.instances import PropertyValueWrite
+from cognite.client.data_classes.data_modeling.views import PropertyType
```

Comment:
nit: should this also be imported from `cognite.client.data_classes.data_modeling.data_types`?